src/_P097_Esp32Touch.ino

```diff
@@ -114,16 +130,25 @@ boolean Plugin_097(uint8_t function, struct EventStruct *event, String& string)
     {
       P097_SEND_TOUCH_EVENT    = isFormItemChecked(F("sendtouch"));
       P097_SEND_RELEASE_EVENT  = isFormItemChecked(F("sendrelease"));
+      //P097_SEND_LONG_PRESS_EVENT = isFormItemChecked(F("sendlongpress"));
+      //P097_LONG_PRESS_TIME = getFormItemInt(F("longpress"));
       P097_SEND_DURATION_EVENT = isFormItemChecked(F("sendduration"));
       P097_TOUCH_THRESHOLD     = getFormItemInt(F("threshold"));
-
+      #if defined(ESP32S2) || defined(ESP32S3)
+      P097_Sleep_WakeUp        = isFormItemChecked(F("sleepwakeup"));
+      #endif
       success = true;
       break;
     }
 
     case PLUGIN_INIT:
     {
       P097_setEventParams(CONFIG_PIN1, P097_TOUCH_THRESHOLD);
+      #if defined(ESP32S2) || defined(ESP32S3)
+      if (P097_Sleep_WakeUp) {
+        touchSleepWakeUpEnable(CONFIG_PIN1, P097_TOUCH_THRESHOLD);
```

Comment:
This is also possible for other ESP32's. At least the ESP32-classic can do this.

Reply:
Ahh ok. I think i misinterpreted the documentation.. will do some testing